obal/data/roles/diff_package/defaults/main.yaml

```diff
@@ -0,0 +1,3 @@
+---
+diff_package_type: brew
+diff_package_skip: true
```

Comment:
This isn't used anymore?

Reply:
I adjusted the -block in main.yml to use this now.